pkgs/applications/editors/kibi/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+with rustPlatform;
+
+buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage rec {
```
Since it's only used once, not really worth using a ``with ...;`` statement.